fact_extractor/unpacker/unpack.py

```diff
@@ -23,7 +23,12 @@ def __init__(self, config=None):
         self._report_folder = Path(self.config.get('unpack', 'data_folder'), 'reports')
 
     def unpack(self, file_path):
-        binary = Path(file_path).read_bytes()
+        compute_stats = self.config.getboolean('ExpertSettings', 'statistics', fallback=True)
+
+        if compute_stats:
+            # Get the data from the initial file to compute statistics after
+            # extracting its content
+            binary = Path(file_path).read_bytes()
```

Comment:
It shouldn't cause any problems to move this down to the other `compute_stats` block (or even inline it since it is only used once) because the file should still be there

Reply:
I moved that piece of code, but ended up keeping it on separate lines as I felt it made it a bit more readable